libraries/WebServer/src/WebServer.cpp

```diff
@@ -551,6 +551,78 @@ void WebServer::enableETag(bool enable, ETagFunction fn) {
   _eTagFunction = fn;
 }
 
+void WebServer::chunkResponseBegin(const char* contentType) {
+  if (_chunkedResponseActive) {
+    log_e("Already in chunked response mode");
+    return;
+  }
+
+  if (strchr(contentType, '\r') || strchr(contentType, '\n')) {
```

Comment:
I think that this is not correct...

`\r` or `\n` can be part of the data to be transfered. 
It just needs to be counted as part of the chunk length.

Chunk Example with `OK\r\n` as content:
```
5\r\n
OK!\r\n\r\n
```